verde/utils.py

```diff
@@ -254,6 +255,72 @@ def grid_to_table(grid):
     return data
 
 
+def load_surfer(fname, dtype='float64'):
+    """
+    Read data from a Surfer ASCII grid file.
+
+    Surfer is a contouring, griding and surface mapping software
+    from GoldenSoftware. The names and logos for Surfer and Golden
+    Software are registered trademarks of Golden Software, Inc.
+
+    http://www.goldensoftware.com/products/surfer
+
+    This feature opens a Surfer grid file, masks any NaNs in the data,
+    and outputs a `xarray.DataArray` that contains easting and northing
```

Comment:
```suggestion
    and outputs a :class:`xarray.DataArray` that contains easting and northing
```

Reply:
This will automatically create a link to the documentation of the DataArray (which I find really handy).